src/Time/Time.js

```diff
@@ -316,6 +315,7 @@ export class Time extends Component {
     super(props);
     const { time } = this.props;
     if (time.hour !== '00') {
+        // do nothing
```

Comment:
This can just be flipped to `if (time.hour == '00')`

Reply:
You know ... I hmmm'd and hah'd over this.  I ended up doing this because of my fear of the undefined, null, etc. values.  But, I may have just been worrying over nothing.  I guess undefined and null would "not equal" `00` so maybe this will be just a quick change.